checker/src/main/java/org/checkerframework/checker/determinism/DeterminismVisitor.java

```diff
@@ -465,9 +465,15 @@ private void checkForDetConditional(ExpressionTree conditionalExpression) {
     }
 
     /**
-     * Reports an error if {@code @PolyDet("up")}, {@code @PolyDet("down")},
-     * {@code @PolyDet("use")}, or {@code @PolyDet("upDet")} is written on a formal parameter or a
-     * return type and none of the formal parameters or the receiver has the type {@code @PolyDet}.
+     * Reports an error if
```

Comment:
This is documenting what the method is doing, but not why it is doing it.  Why is it not legal to but those annotations on formal parameters?